Build/Scripts/BuildLinux.js

```diff
@@ -23,6 +23,9 @@ function copyAtomicNET() {
 
 function copyAtomicEditor() {
 
+    if ( config["noclean"])
+        host.cleanCreateDir(editorAppFolder);
```

Comment:
when I tried the --noclean option, it faulted because the fs.copySync won't overwrite an existing item, Do the other platform --noclean options work this way?